package.json

```diff
@@ -72,14 +74,31 @@
     "@angular/cli": "^13.3.6",
     "@angular/compiler-cli": "^13.3.9",
     "@angular/language-service": "^13.3.9",
+    "@pega/configs": "^0.3.4",
+    "@pega/eslint-config": "^0.3.4",
+    "@pega/prettier-config": "^0.3.4",
     "@playwright/test": "^1.21.1",
     "@types/jasmine": "~3.6.0",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "^13.9.4",
+    "@typescript-eslint/eslint-plugin": "^5.37.0",
+    "@typescript-eslint/parser": "^5.37.0",
     "brotli": "^1.3.2",
     "codelyzer": "^6.0.0",
     "compressing": "^1.5.1",
     "copy-webpack-plugin": "^11.0.0",
+    "eslint": "^8.23.1",
+    "eslint-config-standard-with-typescript": "^23.0.0",
+    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-jest": "^27.0.4",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
+    "eslint-plugin-mdx": "^2.0.5",
+    "eslint-plugin-n": "^15.2.5",
+    "eslint-plugin-prettier": "^4.2.1",
+    "eslint-plugin-promise": "^6.0.1",
+    "eslint-plugin-react": "^7.31.8",
+    "eslint-plugin-react-hooks": "^4.6.0",
```

Comment:
This plugin-react-hooks package would (by its name) seem to be React-oriented. Should we consider removing it in a future PR?